CHANGELOG.unreleased.md

```diff
@@ -30,4 +31,6 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- @xen-orchestra/fs patch
+- vhd-lib patch
```

Comment:
That's good, but do we need to release other packages to fix the problem for users?